docs/topics/configure-build-for-eap.md

```diff
@@ -74,20 +79,25 @@ Here is an example.
 
 For the **kotlinx.coroutines** library, add the version number – `%coroutinesEapVersion%` – that is compatible with `%kotlinEapVersion%`. 
 
-<tabs>
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
 
-```groovy
+```kotlin
 dependencies {
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:%coroutinesEapVersion%"
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:%coroutinesEapVersion%")
 }
 ```
 
-```kotlin
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
 dependencies {
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:%coroutinesEapVersion%")
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:%coroutinesEapVersion%"
```

Comment:
Minor. Seems like we usually use single-quoted strings in Groovy samples. Here are no templates inside, so maybe it is worth changing.

Reply:
I see double quotes were already there, so it is kind of out of this PR context :)

Reply:
Yes, diffs look weird here because I've swapped Groovy and Kotlin code.